### DISCUSSION TITLE
Is the Lua funtion in Yimmenu safe to use?

### DISCUSSION THREAD

**Discussion by fakej1nx1e** [YimMenu/YimMenu#2609] [Q&A] [answered]
My friend coded a Money Loop but i'm scared to use it.

**Comment by Yimura** [comment]
Lua would be no different than in C++, I don’t get your point?

They’re both as safe or _unsafe_ as you make them to be.

**Reply by fakej1nx1e** [reply]
so it depents how safe or unsafe the lua code is?

**Comment by D-Borkan** [comment, marked as answer]
if the money loop is detect so don't use it 
As for the menu, it is safe
